<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscodingParamsWidget</class>
 <widget class="QWidget" name="TranscodingParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="FilenameMask_">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The following variables are allowed in the track filename: $artist, $year, $album, $trackNumber, $title.&lt;/p&gt;&lt;p&gt;Directories are separated by the '/' symbol. File extension is omitted.&lt;/p&gt;&lt;p&gt;For example, a filename could look like this:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;music/$artist/$year - $album/$trackNumber - $artist - $title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The following variables are allowed in the track filename: $artist, $year, $album, $trackNumber, $title.&lt;/p&gt;&lt;p&gt;Directories are separated by the '/' symbol. File extension is omitted.&lt;/p&gt;&lt;p&gt;For example, a filename could look like this:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;music/$artist/$year - $album/$trackNumber - $artist - $title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string notr="true">music/$artist/$year - $album/$trackNumber - $artist - $title</string>
     </property>
     <property name="placeholderText">
      <string>Track filename...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TranscodingBox_">
     <property name="title">
      <string>Transcoding</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="TranscodingFormat_">
        <item>
         <property name="text">
          <string notr="true">ogg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">mp3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quality:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Threads:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="QualitySlider_">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="QualityDisplay_">
          <property name="numDigits">
           <number>3</number>
          </property>
          <property name="intValue" stdset="0">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="ThreadsSlider_">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="ThreadsDisplay_">
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QualitySlider_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QualityDisplay_</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThreadsSlider_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ThreadsDisplay_</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
